File: DSA/Angel_500119630/day2_Question2.java
public class Day2_Question2{
    public static void main(String [] args){
        int [] nums = {1,2,3,4};

        FirstAndLast(nums);
    }
    public static void FirstAndLast(int nums[]){
        int first = nums[0];
        int last = nums[nums.length - 1];

        System.out.println("The first: " + first + ", the last: " + last);
    }
}

File: DSA/pranavdixit_590014418/Day2Question2.py
def first_and_last(nums):
    if not nums:
        return "Array is empty"
    return f"First: {nums[0]}, Last: {nums[-1]}"

# Example usage:
print(first_and_last([1, 2, 3, 4]))   
print(first_and_last([7]))           
print(first_and_last([5, 9, 2]))     
